src/assets/templates/widgets/finaid_summary.html

```diff
@@ -12,6 +13,7 @@ <h2 class="cc-left" style="padding-right: 0">Financial Aid & Scholarships</h2>
         style="margin-bottom: 15px">
     <a class="cc-button cc-button-blue" 
       data-ng-href="finances/finaid/{{financialAidSummary.selected.finaidYear.id}}"
+      data-ng-click="api.analytics.sendEvent('Open new CLC page', 'Click', 'View Awards button')"
```

Comment:
@marcpechaitis  - We've never used `sentEvent` directly like that. We've used [trackExternalLink](trackExternalLink), passing it the name of the current context/card, a string representing the website being linked to, and the URL being linked to.

```javascript
api.analytics.trackExternalLink('Tasks', 'Upload CS Document', task.uploadUrl)
```

Reply:
@redconfetti, ok?  But is that a good enough reason to not use it at all?  In this case, the requirement is not to track an "external link", but instead the behavior of the user... did s/he click the button, or the link?